### ISSUE TITLE
Contents of the  BioNetGen/Models2  directory

### ISSUE THREAD

**Issue by rhclark** [RuleWorld/bionetgen#170]
Sometime soon we should probably have a quick discussion about the contents of the Models2 directory.   I have two files to add, namely  blank_file.bngl  (an empty file)  and  template.bngl  I think that Jim also mentioned that he has come up with some new files for that directory.

**Comment by lh64** [comment]
I don't think that 'Models2' is the proper place for the blank and template files. I think what we want is the ability for a user to create a new file by selecting an option from a drop-down menu somewhere in RB. So these files should probably go in some hidden directory within the RB home directory.
